src/main/index.js

```diff
@@ -27,10 +27,10 @@ process.on('exit', function () {
 
 // Report uncaught exceptions
 process.on('uncaughtException', (err) => {
-  console.error(err)
   const error = { message: err.message, stack: err.stack }
   windows.main.send('uncaughtError', 'main', error)
```

Comment:
Actually, I'm not sure we should even do this. Sending events to the window can cause other uncaught events to happen. I think we should just remove the uncaught handler completely.

Reply:
I'm fine with removing the `windows.main.send()` stuff. But not sure what it does.